app/scripts/directives/qtip.js

```diff
@@ -28,17 +28,17 @@ angular.module('oncokbApp')
                     src = '<iframe width="600px" height="400px" src=\'';
```

Comment:
The component needs some work. It looks like the `type` has been used in most of the cases, and we should define that instead of getting from the attribute. On the line to initialize the qtip, we need to check on which scenario that the qtip will not be initialized?

Reply:
I think `type` is fine because we check its value before binding corresponding behaviors. They won't bother each other.